src/Network/Network/help/New-AzNetworkManagerRoutingRule.md

```diff
@@ -0,0 +1,302 @@
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzNetworkManagerRoutingRule** cmdlet creates a routing rule.
+
+## EXAMPLES
+
+### Example 1: Create Custom routing Rule
+```powershell
+$destination = New-AzNetworkManagerRoutingRuleDestination -DestinationAddress "10.1.1.1/32" -Type "AddressPrefix" 
+$nextHop = New-AzNetworkManagerRoutingRuleNextHop -NextHopType "VirtualAppliance" -NextHopAddress "2.2.2.2"
+New-AzNetworkManagerRoutingRule -ResourceGroupName "psResourceGroup" -NetworkManagerName "psNetworkManager" -ConfigName "psRoutingConfig" -RuleCollectionName "psRuleCollection" -Name "psRule" -Description "TestDescription" -Destination $destination -NextHop $nextHop
+```
+
+```output
+Destination               : {Microsoft.Azure.Commands.Network.Models.NetworkManager.PSNetworkManagerRoutingRuleDestination}
+NextHop                   : {Microsoft.Azure.Commands.Network.Models.NetworkManager.PSNetworkManagerRoutingRuleNextHop}
+DisplayName               :
+Description               : TestDescription
+Type                      : Microsoft.Network/networkManagers/routingConfigurations/ruleCollections/rules
+ProvisioningState         : Succeeded
+SystemData                : Microsoft.Azure.Commands.Network.Models.NetworkManager.PSSystemData
+SystemDataText            : {
+                               "CreatedBy": "00000000-0000-0000-0000-000000000000",
+                               "CreatedByType": "Application",
+                               "CreatedAt": "2021-10-18T04:05:57",
+                               "LastModifiedBy": "00000000-0000-0000-0000-000000000000",
+                               "LastModifiedByType": "Application",
+                               "LastModifiedAt": "2021-10-18T04:05:59"
+                            }
+Name                      : psRule
+Etag                      :
+Id                        : /subscriptions/f0dc2b34-dfad-40e4-83e0-2309fed8d00b/resourceGroups/psResourceGroup/providers/Microsoft.Network/networkManagers/psNetworkManager/routingConfigurations/psRoutingConfig/ruleCollections/psRuleCollection/rules/psRule
+```
+
+Creates a routing rule.
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet in the background
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Description
```

Comment:
Is `Description`  a valid parameter？

Reply:
Yes, it is valid. It was not handled properly before, so I have fixed it now.